tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/api/traversal/FilterTest.java

```diff
@@ -111,6 +110,90 @@ public void test_simple_includes4() {
         Assertions.assertEquals(5, filteredFolderView.assets().versions().size());
     }
 
+    @Test
```

Comment:
I would include a test description to let others know which scenarios are being tested